Initial checkin for FireMon SecurityManager content pack

npatellumeta opened the pull request:
<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

tallieber commented:
hey @npatellumeta can you please fill the [Contribution Registration](https://forms.gle/XDfxU4E61ZwEESSMA) form?

tallieber commented on Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.py:
```diff
@@ -0,0 +1,1078 @@
+        if demisto.command() == 'test-module':
+            # This is the call made when pressing the integration Test button.
+            result = test_module(client)
+            demisto.results(result)
```
We are not using demisto.result() anymore. Use return_results() instead. See documentation [here](https://xsoar.pan.dev/docs/reference/api/common-server-python#return_results)

Please change it in all places demisto.result is used.

tallieber commented on Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.py:
```diff
@@ -0,0 +1,1078 @@
+     """
+    response = client.authenticate_user()
+    if response.get('authorized'):
+        demisto.results("ok")
```
You will use the return_result on the main function, here you just need to return a string "ok"

kgal-pan replied:
resolving as outdated.

tallieber commented on Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.py:
```diff
@@ -0,0 +1,1078 @@
+    if response.get('authorized'):
+        demisto.results("ok")
+    else:
+        demisto.results("Error in API call in FireMonSecurityManager Integrations")
```
```suggestion
        return "Error in API call in FireMonSecurityManager Integrations"
```

kgal-pan replied:
resolving as outdated

tallieber commented on Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.py:
```diff
@@ -0,0 +1,1078 @@
+
+def authenticate_command(client):
+    result = client.authenticate_user().get('token')
+    return result
```
When using `return_result` on main function we need to return from command functions a `CommandResult` object. Docs is [here](https://xsoar.pan.dev/docs/reference/api/common-server-python#commandresults)
for example: 
`    return CommandResults(
        outputs_prefix='AnomaliEnterprise.ForensicSearch',
        outputs_key_field='job_id',
        outputs=start_search_outputs,
        readable_output=tableToMarkdown(name="Forensic search started:", t=start_search_outputs, removeNull=True),
        raw_response=response
    )`

Please change it in all your command functions.

npatellumeta replied:
Is this change a recommendation or required? Since implementing this particular change would touch multiple execution points including playbook, we would like to keep it to what we have. We have also seen the same usage pattern in other integrations as well, so it would be great if we can keep it the way we have and return result instead of CommandResults.

tallieber replied:
If this is xsoar support, this change is required. I saw you didn't change it..

kgal-pan replied:
resolving as outdated.

tallieber commented on Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.py:
```diff
@@ -0,0 +1,1078 @@
+            result = test_module(client)
+            demisto.results(result)
+        elif demisto.command() == 'user-authentication':
+            result = authenticate_command(client)
+            demisto.results(result)
```
```suggestion
            return_results(authenticate_command(client))
```

kgal-pan replied:
resolving as outdated.

tallieber commented on Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.py:
```diff
@@ -0,0 +1,1078 @@
+def main():
+    username = demisto.params().get('credentials').get('identifier')
+    password = demisto.params().get('credentials').get('password')
+    headers = {'Accept': 'application', 'Content-Type': 'application/json'}
```
This variable isn't used in this function

kgal-pan replied:
resolving as outdated

tallieber commented on Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.yml:
```diff
@@ -0,0 +1,137 @@
+  id: FireMonSecurityManager
+  version: -1
+configuration:
+- defaultvalue: https://192.168.21.98
```
You sure this is the default value?

kgal-pan replied:
resolving as outdated

tallieber commented on Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager_test.py:
```diff
@@ -0,0 +1,102 @@
+    requests_mock.post('https://192.168.21.98/securitymanager/api/authentication/login', json=mock_response)
+
+    client = Client(
+        base_url='https://192.168.21.98',
```
you can use here a test url, so it will not failed on validation.

kgal-pan replied:
resolving as outdated

tallieber commented on Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/test_data/get_workflows.json:
```diff
@@ -0,0 +1,805 @@
+                "domainId": 1,
+                "domainName": "Enterprise",
+                "name": "SN Workflow 11.1.8",
+                "pluginGroupId": "com.fm.wf.pp",
```
Please go over all test data to verify there is no secrets or data you dont want him to be public.

kgal-pan replied:
resolving as outdated

tallieber reviewed (changes requested):
Hey, @marcellarichmond @npatellumeta  @mchasepan thanks for your contribution! I review your code, you need to use our new common functions as `Return_results` and `CommandResults`. I wrote you some examples here. If you need any assist here, please don't hesitate to ask.
Also I send you an email to schedule a demo.

tallieber commented on Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.py:
```diff
@@ -0,0 +1,1078 @@
+        elif demisto.command() == 'user-authentication':
+            result = authenticate_command(client)
+            demisto.results(result)
+        elif demisto.command() == 'get-all-workflow':
```
Command names should follow conventions, i.e. vendorname-do-something instead of do-something.

kgal-pan replied:
resolving as outdated

tallieber commented on Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.yml:
```diff
@@ -0,0 +1,137 @@
+    execution: false
+    hidden: false
+    name: pca
+  dockerimage: demisto/python3:3.9.4.18682
```
I think some command are missing here, where is the get-all-workflow

kgal-pan replied:
resolving as outdated

tallieber reviewed (changes requested):
Also, there are files that are missing, the readme file, the pack_metadata file..

npatellumeta commented:
Good afternoon Tal,
 
                We have made all the changes that you have recommended. When we run validate on our code, that is also coming clean. Challenge is; when we try to commit our code pre-commit hook is failing and it is failing on the fixes that we have already made. It looks as if it’s running the hook against checked in code instead of our local code. Is it possible for us to get on a call with you and have a working session to understand what is going on?
 
Here’s the output when we run validate:
 
(venv) namratapatel@Namratas-MacBook-Pro content % demisto-sdk validate -i Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager
You are using demisto-sdk 1.4.9.dev2.
however version 1.5.0 is available.
You should consider upgrading via "pip3 install --upgrade demisto-sdk" command.
 
================= Validating package /Users/namratapatel/xsoar/content/Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager =================
 
Validating /Users/namratapatel/xsoar/content/Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.yml as integration
Validating scheme for /Users/namratapatel/xsoar/content/Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.yml
 
Validating /Users/namratapatel/xsoar/content/Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager_description.md as description
/Users/namratapatel/xsoar/content/Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager_description.md: [DS104] - No detailed description file was found. Consider adding one.
 
 
The files are valid
 
Here’s the output from Git commit:
(venv) namratapatel@Namratas-MacBook-Pro content % git commit -m "Implementing changes from review feedback"                               
Validating files...
You are using demisto-sdk 1.4.9.dev2.
however version 1.5.0 is available.
You should consider upgrading via "pip3 install --upgrade demisto-sdk" command.
 
================= Running validation on branch feature-firemon-securitymanager =================
Validating against origin/master
Running only on staged files
Skipping release notes validation
 
================= Running validation on modified files =================
 
================= Running validation on newly added files =================
 
Validating Packs/FireMonSecurityManager/README.md as readme
There is no node installed on the machine, Test Skipped, error - [Errno 2] No such file or directory: 'node', 
 
Validating Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager_image.png as image
 
Validating Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.yml as integration
Validating scheme for Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.yml
 
Validating Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Create_Policy_Planner_Ticket.yml as playbook
Validating scheme for Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Create_Policy_Planner_Ticket.yml
Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Create_Policy_Planner_Ticket.yml: [ST111] - The field "description" in path 'tasks'-> '0' was not defined in the scheme
To fix the problem, try running `demisto-sdk format -i Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Create_Policy_Planner_Ticket.yml `
 
Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Create_Policy_Planner_Ticket.yml: [ST107] - Missing the field "description" in Path: 'tasks'-> '0'-> 'task'
Please add the field "description" to the path:  'tasks'-> '0'-> 'task' in the yml.
 
Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Create_Policy_Planner_Ticket.yml: [ST111] - The field "description" in path 'tasks'-> '5' was not defined in the scheme
To fix the problem, try running `demisto-sdk format -i Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Create_Policy_Planner_Ticket.yml `
 
Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Create_Policy_Planner_Ticket.yml: [ST107] - Missing the field "description" in Path: 'tasks'-> '5'-> 'task'
Please add the field "description" to the path:  'tasks'-> '5'-> 'task' in the yml.
 
 
Validating Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Pre_Change_Assessment.yml as playbook
Validating scheme for Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Pre_Change_Assessment.yml
Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Pre_Change_Assessment.yml: [ST111] - The field "description" in path 'tasks'-> '0' was not defined in the scheme
To fix the problem, try running `demisto-sdk format -i Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Pre_Change_Assessment.yml `
 
Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Pre_Change_Assessment.yml: [ST107] - Missing the field "description" in Path: 'tasks'-> '0'-> 'task'
Please add the field "description" to the path:  'tasks'-> '0'-> 'task' in the yml.
 
Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Pre_Change_Assessment.yml: [ST111] - The field "description" in path 'tasks'-> '2' was not defined in the scheme
To fix the problem, try running `demisto-sdk format -i Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Pre_Change_Assessment.yml `
 
Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Pre_Change_Assessment.yml: [ST107] - Missing the field "description" in Path: 'tasks'-> '2'-> 'task'
Please add the field "description" to the path:  'tasks'-> '2'-> 'task' in the yml.
 
 
Validating Packs/FireMonSecurityManager/Layouts/layoutscontainer-FireMon_Incident_Pre_Change_Assessment.json as layoutscontainer
Validating scheme for Packs/FireMonSecurityManager/Layouts/layoutscontainer-FireMon_Incident_Pre_Change_Assessment.json
Packs/FireMonSecurityManager/Layouts/layoutscontainer-FireMon_Incident_Pre_Change_Assessment.json: [ST110] - in <SchemaError: error code 2: Schema validation failed:
 - Value 'None' is not a dict. Value path: '/close'.
 - Key 'detached' was not defined. Path: ''.
 - Value 'None' is not a dict. Value path: '/details'.
 - Key 'TypeName' was not defined. Path: '/detailsV2'.
 - Key 'fromServerVersion' was not defined. Path: ''.
 - Value 'None' is not a dict. Value path: '/indicatorsDetails'.
 - Value 'None' is not a dict. Value path: '/indicatorsQuickView'.
 - Key 'itemVersion' was not defined. Path: ''.
 - Key 'locked' was not defined. Path: ''.
 - Value 'None' is not a dict. Value path: '/mobile'.
 - Key 'packID' was not defined. Path: ''.
 - Key 'propagationLabels' was not defined. Path: ''.
 - Value 'None' is not a dict. Value path: '/quickView'.
 - Key 'toServerVersion' was not defined. Path: ''.: Path: '/'>
 
 
Validating Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager_description.md as description
Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager_description.md: [DS104] - No detailed description file was found. Consider adding one.
 
 
Validating Packs/FireMonSecurityManager/Layouts/layoutscontainer-FireMon_Incident_Policy_Planner.json as layoutscontainer
Validating scheme for Packs/FireMonSecurityManager/Layouts/layoutscontainer-FireMon_Incident_Policy_Planner.json
Packs/FireMonSecurityManager/Layouts/layoutscontainer-FireMon_Incident_Policy_Planner.json: [ST108] - The field "detached" was not defined in the scheme
To fix the problem, try running `demisto-sdk format -i Packs/FireMonSecurityManager/Layouts/layoutscontainer-FireMon_Incident_Policy_Planner.json `
 
Packs/FireMonSecurityManager/Layouts/layoutscontainer-FireMon_Incident_Policy_Planner.json: [ST110] - in <SchemaError: error code 2: Schema validation failed:
 - Key 'detached' was not defined. Path: ''.
 - Value 'None' is not a dict. Value path: '/details'.
 - Key 'TypeName' was not defined. Path: '/detailsV2'.
 - Key 'fromServerVersion' was not defined. Path: ''.
 - Value 'None' is not a dict. Value path: '/indicatorsDetails'.
 - Value 'None' is not a dict. Value path: '/indicatorsQuickView'.
 - Key 'itemVersion' was not defined. Path: ''.
 - Key 'locked' was not defined. Path: ''.
 - Value 'None' is not a dict. Value path: '/mobile'.
 - Key 'packID' was not defined. Path: ''.
 - Key 'propagationLabels' was not defined. Path: ''.
 - Key 'toServerVersion' was not defined. Path: ''.: Path: '/'>
 
 
================= Running validation on changed pack unique files =================
 
Validating Packs/FireMonSecurityManager unique pack files
 
Running pack dependencies validation on FireMonSecurityManager
 
Found first level dependencies only on core packs
 
=========== Found errors in the following files ===========
 
Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Create_Policy_Planner_Ticket.yml - [ST111]
Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Create_Policy_Planner_Ticket.yml - [ST107]
Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Pre_Change_Assessment.yml - [ST111]
Packs/FireMonSecurityManager/Playbooks/playbook-FireMon_Pre_Change_Assessment.yml - [ST107]
Packs/FireMonSecurityManager/Layouts/layoutscontainer-FireMon_Incident_Pre_Change_Assessment.json - [ST110]
Packs/FireMonSecurityManager/Layouts/layoutscontainer-FireMon_Incident_Policy_Planner.json - [ST108]
Packs/FireMonSecurityManager/Layouts/layoutscontainer-FireMon_Incident_Policy_Planner.json - [ST110]
 
The files were found as invalid, the exact error message can be located above
You are using demisto-sdk 1.4.9.dev2.
however version 1.5.0 is available.
You should consider upgrading via "pip3 install --upgrade demisto-sdk" command.
Starting secrets detection
Running secrets validation on all changes
Finished validating secrets, no secrets were found.
 
Skipping running lint tasks (flake8, mypy, pylint, pytest). If you want to run this as part of the precommit hook
set CONTENT_PRECOMMIT_RUN_DEV_TASKS=1. You can add the following line to ~/.zshrc:
echo "export CONTENT_PRECOMMIT_RUN_DEV_TASKS=1" >> ~/.zshrc
 
Or if you want to manually run lint tasks: demisto-sdk lint -i <integration/script dir>
Example: demisto-sdk lint -i Scripts/ParseEmailFiles
Please fix the aforementioned errors and then commit again
 
Thanks,
Namrata

tallieber commented:
@npatellumeta Hey, you can try commit the files without the pre-commit hooks if you run all the validations before. ` git commit -n -m "Implementing changes from review feedback"`

npatellumeta commented:
Tal,

                Thank you so much. We have just pushed our code without the pre-commit hook.

Namrata Patel
Development Manager | FireMon
D: 732.357.3526 | M: 908.672.0721
***@***.******@***.***> | www.firemon.com<http://www.firemon.com/>


From: Tal Lieber ***@***.***>
Reply-To: demisto/content ***@***.***>
Date: Wednesday, September 29, 2021 at 6:49 AM
To: demisto/content ***@***.***>
Cc: Namrata Patel ***@***.***>, Mention ***@***.***>
Subject: Re: [demisto/content] Initial checkin for FireMon SecurityManager content pack (#14560)

CAUTION: This email originated from outside of FireMon. Do not click links or open attachments unless you recognize the sender and know the content is safe.

@npatellumeta<https://github.com/npatellumeta> Hey, you can try commit the files without the pre-commit hooks if you run all the validations before. git commit -n -m "Implementing changes from review feedback"

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/demisto/content/pull/14560#issuecomment-930063749>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ADB4QWZWEQR7QBGRDMCW26DUELVKHANCNFSM5C5SNETA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

tallieber commented on Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.py:
```diff
@@ -0,0 +1,324 @@
+        elif demisto.command() == 'user-authentication':
+            return_results(authenticate_command(client))
+        elif demisto.command() == 'create-pp-ticket':
+            return_results(create_pp_ticket_command(client, demisto.args()))
```
When using return_result on main function we need to return from command functions, e.g `create_pp_ticket_command` a CommandResult object. Docs is here
for example:
return CommandResults( outputs_prefix='AnomaliEnterprise.ForensicSearch', outputs_key_field='job_id', outputs=start_search_outputs, readable_output=tableToMarkdown(name="Forensic search started:", t=start_search_outputs, removeNull=True), raw_response=response )

tallieber commented:
@npatellumeta also pay attention for the validations - there are still some secrets you should remove. Regarding the unit tests failures- you need to change the import

tallieber commented on Packs/FireMonSecurityManager/pack_metadata.json:
```diff
@@ -0,0 +1,16 @@
+{
+    "name": "FireMon Security Manager",
+    "description": "Creates a Policy Planner Ticket in Policy Planner Application and Validates Pre Changes assessments for new requirement.",
+    "support": "xsoar",
```
The support is parter right? 
Need to change the support, url and author to contain details regarding the partner.

npatellumeta commented:
Would someone from XSOAR team resolve the conflict for Tests/conf.json? It looks like because of this, LGTM analysis tasks are failing.

npatellumeta commented:
We have also accepted and signed Contributor License Agreement. not sure why it is showing up as pending

kgal-pan commented:
@npatellumeta - Looks like the branch is 10 commits behind `master`. Can you please merge from `master`? 

This will help resolve the conflict for `Tests/conf.json`:

```bash
git checkout master

git pull

git checkout <your branch_name>

git merge master
```

npatellumeta commented:
When tried this earlier and just tried it again. It seems to think that there’s nothing to merge. Please see below snippet:





(venv) ***@***.*** content % git checkout master

Switched to branch 'master'

Your branch is up to date with 'origin/master'.

(venv) ***@***.*** content % git pull

Already up to date.

(venv) ***@***.*** content % git status

git On branch master

Your branch is up to date with 'origin/master'.



nothing to commit, working tree clean

(venv) ***@***.*** content % git checkout feature-firemon-securitymanager

Switched to branch 'feature-firemon-securitymanager'

(venv) ***@***.*** content % git merge master

Already up to date.

(venv) ***@***.*** content % git status

On branch feature-firemon-securitymanager

nothing to commit, working tree clean

Thanks,
Namrata Patel
Development Manager | FireMon
D: 732.357.3526 | M: 908.672.0721
***@***.******@***.***> | www.firemon.com<http://www.firemon.com/>


From: Kobbi Gal ***@***.***>
Reply-To: demisto/content ***@***.***>
Date: Monday, October 4, 2021 at 9:25 AM
To: demisto/content ***@***.***>
Cc: Namrata Patel ***@***.***>, Mention ***@***.***>
Subject: Re: [demisto/content] Initial checkin for FireMon SecurityManager content pack (#14560)

CAUTION: This email originated from outside of FireMon. Do not click links or open attachments unless you recognize the sender and know the content is safe.

@npatellumeta<https://github.com/npatellumeta> - Looks like the branch is 10 commits behind master. Can you please merge from master?

This will help resolve the conflict for Tests/conf.json:

git checkout master



git pull



git checkout <your branch_name>



git merge master

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/demisto/content/pull/14560#issuecomment-933484281>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ADB4QW5LMIWYP2NOE4CAXZDUFGTNLANCNFSM5C5SNETA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

tallieber commented on Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.py:
```diff
@@ -0,0 +1,365 @@
+        if demisto.command() == 'test-module':
+            result = test_module(client)
+            demisto.results(result)
+        elif demisto.command() == 'user-authentication':
+            return_results(authenticate_command(client))
+        elif demisto.command() == 'create-pp-ticket':
+            return_results(create_pp_ticket_command(client, demisto.args()))
+        elif demisto.command() == 'pca':
+            return_results(pca_command(client, demisto.args()))
```
Command names should follow conventions, i.e. vendorname-do-something instead of do-something (with the exception of reputation commands)

Please change it, for example - `firemon-pac`, `firemon-create-pp-ticket`

kgal-pan replied:
See https://xsoar.pan.dev/docs/integrations/code-conventions.

tallieber reviewed (changes requested):
change the command name on py, yml and readme

tallieber commented on Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/Tests/id_set.json:
```diff
@@ -0,0 +1,21 @@
+{
+    "scripts": [],
+    "playbooks": [],
+    "integrations": [],
+    "TestPlaybooks": [],
+    "Classifiers": [],
+    "Dashboards": [],
+    "IncidentFields": [],
+    "IncidentTypes": [],
```
Does this file is required? 
This seems empty..

npatellumeta replied:
We have deleted this file

tallieber commented on Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/test_data/Tests/id_set.json:
```diff
@@ -0,0 +1,21 @@
+{
+    "scripts": [],
+    "playbooks": [],
+    "integrations": [],
+    "TestPlaybooks": [],
+    "Classifiers": [],
+    "Dashboards": [],
+    "IncidentFields": [],
+    "IncidentTypes": [],
+    "IndicatorFields": [],
+    "IndicatorTypes": [],
+    "Layouts": [],
+    "Reports": [],
+    "Widgets": [],
+    "Mappers": [],
+    "Packs": {},
+    "GenericTypes": [],
+    "GenericFields": [],
+    "GenericModules": [],
+    "GenericDefinitions": []
+}
```
Does this file is required?
This seems empty..

tallieber commented on Packs/FireMonSecurityManager/secrets_white_list.json:
```diff
@@ -0,0 +1,12 @@
+{
+  "iocs": {
+    "ips": [
+    ],
+    "emails": [
+      "test@example.com"
+    ],
```
Do you use this file?

npatellumeta replied:
Yes we are using this file

npatellumeta commented:
Canceling PR to resolve multiple username commit issue which is keeping CLA check from passing

npatellumeta commented:
Since we were running into issue with CLA agreement because of multiple committers, we have created a new PR (https://github.com/demisto/content/pull/15584) to submit our integration. Closing this as recommended by @marcellarichmond and @ShahafBenYakir
